doc/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -43,13 +57,14 @@ We will have to generate a YAML configuration file for configuring the 3D sensor
 
 
 YAML Configuration file (Depth Map)
------------------------------------
++++++++++++++++++++++++++++++++++++
 
-We will have to generate a YAML configuration file for configuring the 3D sensors. An `example file for processing depth images <https://github.com/ros-planning/panda_moveit_config/blob/master/config/sensors_kinect_depthmap.yaml>`_ can be found in the `panda_moveit_config repository <https://github.com/ros-planning/panda_moveit_config>`_ as well: ::
+We will have to generate a YAML configuration file for configuring the 3D sensors. An `example file for processing depth images <https://github.com/ros-planning/panda_moveit_config/blob/master/config/sensors_kinect_depthmap.yaml>`_ can be found in the `panda_moveit_config repository <https://github.com/ros-planning/panda_moveit_config>`_ as well.
```

Comment:
This is something I want to ask about since I'm not sure, is it possible to have both plugins working at the same time?

Reply:
i don't think so but im not 100% on that